test/integration/aws/default/verify/controls/aws_flow_log.rb

```diff
@@ -0,0 +1,44 @@
+fixtures = {}
+%w[fl-vpc fl-subnet subnet_01_id ec2_security_group_default_vpc_id].each do |fixture_name|
```

Comment:
If you need ``ec2_security_group_default_vpc_id`, you should export it from the flow-logs.tf, and give it a flow-logs specific name.